peyvandak/Majlesi/django-docker-compose-url-shortener/urlshortener_api/api/views.py

```diff
@@ -0,0 +1,177 @@
+from .models import Shortener, VisitUrl
+from django.http import JsonResponse, HttpResponseRedirect, Http404
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+from django.contrib.auth.models import User
+from django.contrib.auth import authenticate, login
+from django.contrib.auth import logout
```

Comment:
it would have been better if you merged line 7 and 8 like this:
from django.contrib.auth import authenticate, login, logout